package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "parsimmon",
-  "version": "1.16.0",
+  "version": "1.17.0",
```

Comment:
`npm version` will update this for me, and it gets mad if the version number is already there in package.json. please revert this part and then i can merge once the tests pass :)

Reply:
My bad, reverted

Reply:
it's all good. i make this mistake all the time